sdk/schemaregistry/azure-schemaregistry/samples/async_samples/sample_code_schemaregistry_async.py

```diff
@@ -27,31 +27,31 @@
 import asyncio
 
 from azure.schemaregistry.aio import SchemaRegistryClient
-from azure.schemaregistry import SerializationType
+from azure.schemaregistry import SchemaFormat
 from azure.identity.aio import ClientSecretCredential, DefaultAzureCredential
 
 
 def create_client():
     # [START create_sr_client_async]
-    SCHEMA_REGISTRY_ENDPOINT = os.environ['SCHEMA_REGISTRY_ENDPOINT']
+    SCHEMA_REGISTRY_FQN = os.environ['SCHEMA_REGISTRY_ENDPOINT']
```

Comment:
I think we also need to update the `os.environ['SCHEMA_REGISTRY_ENDPOINT'] to be `['SCHEMA_REGISTRY_FULLY_QUALIFED_NAMESPACE']` in the samples to stay consistent with the term we choose.

Reply:
yeah that makes sense, I was just lazy about it :p. Good catch!